libs/perun/pipes/src/lib/get-dialog-type.pipe.ts

```diff
@@ -0,0 +1,17 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
+
+@Pipe({
+  name: 'getDialogType',
```

Comment:
The naming convention we have is, that there is no "get" in pipe names, just the "purpose" e.g. `userFullName` pipe.